.travis.yml

```diff
@@ -22,11 +22,9 @@ install:
   - make setup
 
 env:
-  - RUN="make lint"
-  - RUN="make test-unit"
+  - RUN="make test-unit test-examples lint coveralls"
```

Comment:
There was a small discussion about linting with @prashantv in https://github.com/uber/ringpop-go/issues/160 where he explained why the linter was not ran against the examples always. The TLDR is iirc that the linter runs against the object code not the source.

By running it at the end we are ensured that all relevant objects have been made, thats why I put it at the end.